geoips/schema/plugin_validator.py

```diff
@@ -46,11 +46,18 @@ def get_all_schema():
     schema_path = files("geoips.schema")
     # Don't use `schema_path.glob` since if it is a MultiplexedPath it won't have `glob`
     schema_files = glob(str(schema_path / "*.yaml"))
+    schema_files += glob(str(schema_path / "*/*.yaml"))
 
     all_schema = {}
     for schema_file in schema_files:
-        schema_name = os.path.splitext(os.path.basename(schema_file))[0]
         schema = yaml.safe_load(open(schema_file, "r"))
+        schema_name = (
```

Comment:
Note we may want to update this to be prettier... I just hacked up the path to get interface.family from the schema file.